File: PYTHON/corriente media es mayor a 3.py
corriente = float(input("Ingrese corriente (A): "))
voltaje = float(input("Ingrese voltaje (V): "))

if corriente > 3 and voltaje < 1:
    print("Alarma de CORTOCIRCUITO")
else:
    print("Sistema en monitoreo...")

File: C/corriente media es mayor a 3.c
#include <stdio.h>

int main() {
    float corriente, voltaje;
    printf("Ingrese corriente (A): ");
    scanf("%f", &corriente);
    printf("Ingrese voltaje (V): ");
    scanf("%f", &voltaje);

    if (corriente > 3 && voltaje < 1) {
        printf("Alarma de CORTOCIRCUITO\n");
    } else {
        printf("Sistema en monitoreo...\n");
    }

    return 0;
}
